includes/application/handleAppl.php

```diff
@@ -159,7 +159,26 @@
 		
 		$curDate = date('Y-m-d'); 
 		
-		if( ( strtotime($curDate) - strtotime($genDate) ) > 10*24*60*60 )
+		$sqlconn = new mysqli ( $eorgDBserver, $eorgDBuser, $eorgDBpasswd, $eorgDBname );
+
+		if ( $sqlconn->connect_errno )
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Expected 0 spaces after opening bracket; 1 found
* Expected 0 spaces before closing bracket; 1 found
* Expected 1 space after closing parenthesis; found 9

<!-- Comment by [SideCI](https://sideci.com) -->